qa-tests-backend/src/main/groovy/services/ImageService.groovy

```diff
@@ -37,11 +39,14 @@ class ImageService extends BaseService {
 
     static scanImage(String image, Boolean includeSnoozed = true, Boolean force = false) {
         try {
-            return getImageClient().scanImage(ImageServiceOuterClass.ScanImageRequest.newBuilder()
-                    .setImageName(image)
-                    .setIncludeSnoozed(includeSnoozed)
-                    .setForce(force)
-                    .build())
+            def imageClient = getImageClient().scanImage(ImageServiceOuterClass.ScanImageRequest.newBuilder());
+            return withRetry(1, 15) {
+                return imageClient
+                        .setImageName(image)
+                        .setIncludeSnoozed(includeSnoozed)
+                        .setForce(force)
+                        .build()
+            };
```

Comment:
Seeing this error in the test output:
```
VulnMgmtTest STANDARD_OUT
    03:07:20 | INFO  | VulnMgmtTest              | Starting testsuite
    03:07:20 | ERROR | ImageService              | Image failed to scan: quay.io/rhacs-eng/qa:ansibleplaybookbundle--gluster-s3object-apb--481960439934084fb041431f27cb98b89666e1a0daaeb2078bcbe1209790368c
    groovy.lang.MissingMethodException: No signature of method: io.stackrox.proto.api.v1.ImageServiceGrpc$ImageServiceBlockingStub.scanImage() is applicable for argument types: (io.stackrox.proto.api.v1.ImageServiceOuterClass$ScanImageRequest$Builder) values: []
    Possible solutions: scanImage(io.stackrox.proto.api.v1.ImageServiceOuterClass$ScanImageRequest), getImage(io.stackrox.proto.api.v1.ImageServiceOuterClass$GetImageRequest)
    	at services.ImageService.scanImage(ImageService.groovy:42) [3 skipped]
    	at services.ImageService.scanImage(ImageService.groovy)
```
May need to shift things around, perhaps something like this:

```suggestion
def imageReq = ImageServiceOuterClass.ScanImageRequest.newBuilder()
                .setImageName(image)
                .setIncludeSnoozed(includeSnoozed)
                .setForce(force)
                .build()
                    
return withRetry(1, 15) {
    return getImageClient().scanImage(req)
};
```